Libraries/Components/Picker/Picker.js

```diff
@@ -166,24 +166,24 @@ class Picker extends React.Component<PickerProps> {
 
   static Item: typeof PickerItem = PickerItem;
 
-  static defaultProps: {|mode: $TEMPORARY$string<'dialog'>|} = {
-    mode: MODE_DIALOG,
-  };
-
   render(): React.Node {
+    const {mode = MODE_DIALOG, ...rest} = this.props;
+
     if (Platform.OS === 'ios') {
       /* $FlowFixMe[prop-missing] (>=0.81.0 site=react_native_ios_fb) This
        * suppression was added when renaming suppression sites. */
       /* $FlowFixMe[incompatible-type] (>=0.81.0 site=react_native_ios_fb) This
        * suppression was added when renaming suppression sites. */
-      return <PickerIOS {...this.props}>{this.props.children}</PickerIOS>;
+      return <PickerIOS {...{mode, ...rest}}>{this.props.children}</PickerIOS>;
```

Comment:
I'd be tempted to make this simpler and without the extra object creation
```
mode={mode}
{...rest}
```

Reply:
I have tried this way, but it will throw error of flow type.

Reply:
Interesting... what's the flow error?